### PR TITLE
Clean 01 deploy to base

### PR DESCRIPTION
**Pull request by reespozzi**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: apps/admin/ithc/01/kustomization.yaml [hunk 1]
```diff
@@ -7,5 +7,3 @@ resources:
 
 patchesStrategicMerge:
   - ../../traefik2/ithc/01.yaml
-  - ../../traefik2/ithc/secret-provider.yaml
```
Comment (thomast1906) [inline apps/admin/ithc/01/kustomization.yaml:10]: This also needs removed from 01
https://github.com/hmcts/cnp-flux-config/blob/clean-01-deploy-to-base/apps/admin/ithc/01/kustomization.yaml#L5

Comment (thomast1906) [reply apps/admin/ithc/01/kustomization.yaml:10]: & L6

Comment (thomast1906) [reply apps/admin/ithc/01/kustomization.yaml:10]: & could you remove any traefikv1 files that aren't needed now for ithc?

**Comment by reespozzi** [comment]
https://github.com/hmcts/cnp-flux-config/pull/16918

**Pull request closed without merging**